### ISSUE TITLE
[V2] Invalid query string param ordering for signature

### ISSUE THREAD

**Issue by discosultan** [sammchardy/python-kucoin#42]
As of KuCoin API v2, the query string parameters used in signature generation should be ordered as specified in docs and not alphabetically.

Essentially the `sorted()` function call here is incorrect: https://github.com/sammchardy/python-kucoin/blob/3f98f166cad3096c7e376c9ffb1a2849a55822ea/kucoin/client.py#L99

To reproduce:

```py
from kucoin.client import Client

client = Client('<key>', '<secret>', '<passphrase>')
client.get_orders(status='done', symbol='ETH-BTC')
```

**Comment by sammchardy** [comment]
Thanks @discosultan can you check if this is fixed in v2.0.2

**Comment by discosultan** [comment]
@sammchardy That fixed it for me, thank you!
